final_prep/README.md

```diff
@@ -19,37 +19,37 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![Screen Shot 2021-08-07 at 10 24 57 AM](https://user-images.githubusercontent.com/81737385/128607045-e91bbb43-80e0-4e22-9e70-883c7ac24f50.png)
+- ![Screen Shot 2021-08-07 at 10 25 32 AM](https://user-images.githubusercontent.com/81737385/128607050-a1b787da-9b0e-4a14-b81b-14eb20528a86.png)
```

Comment:
Good job overall, but I'd love to see more of that pinkish color scattered through the week! Also, you'll have to evaluate how long work time blocks are going on the weekends. I needed to get more specific with scheduling for particular tasks at hand to help keep me focused after a draining week. You'll get more of a feel for your patterns as you start Turing, just remember to try to be honest with yourself, and take time weekly to work on adjusting your calendar!